manifests/agent.pp

```diff
@@ -142,6 +154,30 @@
     notify    => Service['sensu-agent'],
   }
 
+  if $service_env_vars_file {
+    file { 'sensu-agent_env_vars':
+      ensure    => 'file',
+      path      => $service_env_vars_file,
+      content   => join($_service_env_vars_content, "\n"),
+      owner     => $::sensu::sensu_user,
+      group     => $::sensu::sensu_group,
+      mode      => $::sensu::file_mode,
+      show_diff => $show_diff,
+      require   => Package['sensu-go-agent'],
+      notify    => Service['sensu-agent'],
+    }
+  }
+  if $facts['os']['family'] == 'windows' {
```

Comment:
Does windows not support reading the values from a file like the other platforms? If so, that seems like a bug in sensu-go.

Reply:
I think this is Windows issue because the application reading values from file is systemd or sysVinit on Linux. All sensu-go does is read environment variables.

Reply:
perhaps just a comment that sensu-go does not read environment variables from a file on windows

Reply:
comment added